app/models/location.rb

```diff
@@ -1,5 +1,37 @@
+require 'geokit'
+
 class Location < ActiveRecord::Base
   belongs_to :user
 
   validates :user, presence: true
+
+  after_save :reindex_dishes
+
+  def reindex_dishes
+    user.dishes.each { |dish| dish.reindex }
```

Comment:
@xudongcamsys, continuing on from our earlier conversation about this section of the README https://github.com/ankane/searchkick#reference , this might help with line 11:
```ruby
  Dish.searchkick_index.import(user.dishes)
```